src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -0,0 +1,123 @@
+const {
+  getDrupalValue,
+  utcToEpochTime,
+  getWysiwygString,
+  uriToUrl,
+} = require('./helpers');
+const { mapKeys, camelCase } = require('lodash');
+const assert = require('assert');
+
+function fakeUtc(timeString) {
+  // Assume the timeString looks like 2019-05-30T21:00:00
+  // If it doesn't, we'll need to do some real conversions; fail for now
+  assert(
+    /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}$/.test(timeString),
+    `Expected timeString to look like 2019-05-30T21:00:00, but found ${timeString}`,
+  );
+  return `${timeString.replace('T', ' ')} UTC`;
+}
+
+// Same as the createMetaTag in the transformer helpers, but uses
+// __typename instead of type. Because consistency.
+function createMetaTag(type, key, value) {
+  return {
+    __typename: type,
+    key,
+    value,
+  };
+}
+
+const transform = entity => {
+  const metaTags = entity.metatag.value;
+
+  return {
+    entityType: 'node',
+    entityBundle: 'event',
+    title: getDrupalValue(entity.title),
+    uid: entity.uid[0],
+    changed: utcToEpochTime(getDrupalValue(entity.changed)),
+    entityUrl: {
+      // TODO: Get the breadcrumb from the CMS export when it's available
+      breadcrumb: [],
+      path: entity.path[0].alias,
+    },
+    entityMetatags: [
```

Comment:
Also worth noting: In `node-page`, this is `entityMetaTags`, but here it's `entityMetatags` with a lowercase `t`. This is intentional.

And bothersome.

Reply:
:sob: 

Maybe it is worth looking into the templates to see if there is a reason for that?

Reply:
:man_shrugging: It's probably easy enough to refactor, but I think out of scope for now since we can't change the templates while we're still getting data from the GQL query. (We might could change the query itself, but that's more work than I want to do right now. That's refactoring work, not creation work like we're doing now.)

Reply:
Oh yeah, I wasn't suggesting that you do that as part of this PR - just an observation for potentially _more_ future work on this stuff :wink: